### ISSUE TITLE
[Feature Request] v-rating Rules Validation

### ISSUE THREAD

**Issue by melvinchng** [vuetifyjs/vuetify#11163]
### Problem to solve
Unable to validate whether the v-rating question has been answered by the user

### Proposed solution
Accepts an array of functions that take an input value as an argument and return either true / false or a string with an error message

**Comment by AntoninBeaufort** [comment]
It's a good idea, I need this too.

**Comment by a-t** [comment]
You can just use the [`v-input`](https://vuetifyjs.com/en/components/inputs/) as a wrapper to add validation:
```html
<v-input :value="value" :rules="rules">
  <template>
    <v-rating v-model="value" />
  </template>
</v-input>
```

**Comment by Tofandel** [comment]
Got surprised it wasn't there by default.. It's an input so it should have it

**Comment by johnleider** [comment]
resolved by #14026

This can now be done using the `v-validation` component.

```html
<v-validation :rules="[rules]">
  <template #default="{ validate, errorMessages, isValid, ...etc }">
    <v-rating @blur="validate" v-model="model" />

    <v-messages :active="isValid.value" :value="errorMessages.value" />
  </template>
</v-validation>
```

If you have any questions, please reach out to us in our [Discord community](https://community.vuetifyjs.com).
